ambry-store/src/test/java/com.github.ambry.store/IndexValueTest.java

```diff
@@ -213,7 +213,7 @@ private void verifyGetters(IndexValue value, String logSegmentName, long size, l
     }
     assertEquals("Original message offset not as expected", originalMessageOffset, value.getOriginalMessageOffset());
     if (version == 1) {
-      assertEquals("Operation time mismatch", Math.toIntExact(operationTimeInSecs), value.getOperationTimeInSecs());
+      assertEquals("Operation time mismatch", (int) operationTimeInSecs, value.getOperationTimeInSecs());
```

Comment:
yes, trying to find a way for it. For now, this works upto value of 2147483647999 which is (Integer.MAX_VALUE * 1000) -1. This translates to Tue Jan 19 2038 03:14:07. If someone sets an expiry after this date, this will break. Will get back to you once I have a good solution.